bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Table.java

```diff
@@ -1479,10 +1479,10 @@ public void clearAll () {
 	int border = getBorderWidthInPixels ();
 	width += border * 2;  height += border * 2;
 	if ((style & SWT.V_SCROLL) != 0) {
-		width += OS.GetSystemMetrics (OS.SM_CXVSCROLL);
+		width += getSystemMetrics (OS.SM_CYHSCROLL);
```

Comment:
I think the parameter is still wrong
```suggestion
		width += getSystemMetrics (OS.SM_CXVSCROLL);
```

Reply:
Sorry for that.